Psychtoolbox/PsychHardware/NetStation.m

```diff
@@ -183,7 +213,53 @@
                 if n >= 100
                     warning('\nNetStationPC synchronization did not succeed within %.1f ms\nSynchronizatoin accuracy is %.1f ms\n',NSSynchLimit,df);
                 end
+                GETNTPSYNCED = 0;
+            end
+
+        %%% added by Andreas Widmann, widmann@uni-leipzig.de, 2017-07-19
+        case 'getntpsynchronize'
+            
+            if isempty(NSIDENTIFIER) || (NSIDENTIFIER < 0)
+                status = 1;
+            else
+                
+                % Open UDP socket for NTP server connection
+                if isempty( NTPSOCK )
+                    if nargin < 2 || isempty( varargin { 2 } )
+                        error( 'NTP server host name required.' )
+                    end
+                    NTPSOCK = GetNTP( 'open', varargin{ 2 } );
+                end
+                
+                % Get initial drift estimate
+                NTPLOG = [];
+                fprintf( 'Estimating drift...\n' );
```

Comment:
Maybe add a NetStation-Info: prefix, so no random messages about some unknown entity estimating some unknown drift show up in the console or logs.